new_test_page.txt

```diff
@@ -6,3 +6,5 @@ The whole purpose of the orphantest project was to see if I could create an orph
 Well, the great news is yes, you can do all of that with an orphan branch, and basically have an alternate project alongside the main project in the same repo, or overwrite the Git history of a project with this branch by deleting all branches and files that are branched off of the original main/master branch.
 
 And also, there are other great things that can come with this.  For instance, if you create an orphan branch, but do not know yet what to put into the branch, you can use `git commit --allow-empty -m <message>` to initialize the branch without the need of a file!  How cool is that?  I wonder if you could do the same thing when you create and initialize a whole new repo and start on main/master.  I also wonder if you can do that for any time you start off on a new branch from another branch, and you just want to state that you created another branch before doing anything else with the content there.   I think teaching people that command and bringing that command to light would be such a great idea for future Git and GitHub tutorials.
+
+Judging solely from my experience now, one best practice I would suggest for anyone who has a repo with orphan branches is to use the orphan branch where all content is based, and use that as a prefix to other branches from that orphan branch, so that repo owners, integrators, and administrators know which project each person is working on while being able to have all projects in one repo.
```

Comment:
Great point in starting to think about best practices with orphan branches.  I have heard about this concept, and I agree, if there is no way to organize the branches, then how will we know what exact project our contributors are working on, if we were to consolidate multiple projects into one repo?

Reply:
Well, thank you @astronomical3.